.github/workflows/build.yml

```
@@ -140,9 +140,16 @@ jobs:
               `;
             }, '' );
             const commentBody = `${ TEST_COMMENT_INDICATOR }
-            Are you an Automattician? You can now test your Pull Request on WordPress.com. On your sandbox, run
+            Are you an Automattician? Please test your changes on WordPress.com.
+
+            - To test on WoA, install [the Beta plugin](https://jetpack.com/download-jetpack-beta/), go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
```

```suggestion
            - To test on a WoA site: Create a new Business site and take it Atomic, install and activate [the Jetpack Beta plugin](https://jetpack.com/download-jetpack-beta/), go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
```

---

>  Create a new Business site and take it Atomic

I worry this may be a bit cryptic for some. What do you think about 

```suggestion
            - To test on WoA, go to the Plugins menu on a WordPress.com Simple site. Click on the "Upload" button and follow the upgrade flow to be able to upload, install, and activate [the Jetpack plugin](https://jetpack.com/download-jetpack-beta/). Once the plugin is active, go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
```

---

@jeherve I think that works too 👍